scripts/build.sh

```diff
@@ -38,6 +38,11 @@ for bin in *; do
     set -x
     # -race requires CGO
     CGO_ENABLED=1 go build -race -ldflags "-X main.version=$version" -o $BUILDDIR/$bin || fail
+  elif [ "$1" == "-debug" ]
+  then
+    set -x
+    # -debug flags
+    CGO_ENABLED=0 go build -gcflags "all=-N -l" -ldflags "-X main.version=${version}-debug" -o $BUILDDIR/$bin || fail
```

Comment:
are race detector and dlv mutually exclusive?
seems both should work at the same time.
if so, perhaps we should always build the debug images?

Reply:
> and only one binary is produced from build.sh 
this is incidental. not by design.
everything else you say here seems fair enough.